medic/medic-log.js

```diff
@@ -66,7 +66,7 @@ function logIOS() {
 function logWindows() {
     var logScriptPath = path.join("mobilespec", "platforms", "windows", "cordova", "log.bat");
     if (fs.existsSync(logScriptPath)) {
-        shelljs.exec(logScriptPath + " --dump", function (code, output) {
+        shelljs.exec(logScriptPath + " --dump --mins 15", function (code, output) {
```

Comment:
I know this is already merged, but perhaps you can pass the build timeout as a parameter, so we always wait longer than the timeout even if the timeout is increased later.